### DISCUSSION TITLE
JSX Pages sometimes can partially render the HTML

### DISCUSSION THREAD

**Discussion by tisonkun** [facebook/docusaurus#10077] [Q&A] [answered]
The homepage of Apache StreamPark (https://streampark.apache.org/) can render HTML with the footer, including "Copyright © 2022-2024 The Apache Software Foundation.".

But the other page also generated by JSX (https://streampark.apache.org/team) is purely JS-based.

Their sources can be found at:

* https://github.com/apache/incubator-streampark-website/blob/dev/src/pages/index.tsx
* https://github.com/apache/incubator-streampark-website/blob/dev/src/pages/team/index.jsx

The one that partially rendered, i.e., has the footer in the HTML, is desired. I wonder how I can implement it for:

* https://streampark.apache.org/team
* https://streampark.apache.org/user

I try to rename `team/index.jsx` with `.tsx` and try to remove the `BrowserOnly` wrapper, but none of them works.

**Comment by Josh-Cena** [comment]
How does it not work if you remove `<BrowserOnly>`? You can put the `AOS.init` part inside a `useEffect`

**Reply by tisonkun** [reply]
As stated above:

> try to remove the BrowserOnly wrapper, but none of them works.

**Reply by tisonkun** [reply, marked as answer]
Seems switch the order or `BrowserOnly` and `Layout` works. Just keep `Layout` the most outer page.

**Reply by tisonkun** [reply]
And I suppose remove `BrowserOnly` should work also. I confirm the switch place solution work by invalid browser cache. It seems the cache keeps return the unrenderred page source.
